test_schemas/en/test_routing_checkbox_count.json

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
Worth prefixing with `test_new_routing` to make deprecation of the old one easier.